tools/make/lib/init/Makefile

```diff
@@ -36,5 +36,6 @@ include $(TOOLS_MAKE_LIB_DIR)/init/git_hooks.mk
 # make init
 #/
 init: init-git-hooks init-eslint-plugins
+	@echo "Initialization Successful."
```

Comment:
@Planeshifter This should be `$(QUIET) echo "..."`

Reply:
I addressed in a follow-up commit.

Reply:
Thanks for correcting.